### PR TITLE
Create comment model

### LINKED ISSUE #12: Comment model
fields:
- content
- authorId

Blocked by #4 #3

### PR DESCRIPTION
**Pull request by Darnoker**
Closes #12

### CODE REVIEW

File: ugbook-back/src/schemas/comment.schema.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { Prop, Schema, SchemaFactory, unique } from '@nestjs/mongoose';
+import { Document } from 'mongoose';
+
+export type CommentDocument = Comment & Document;
+
+@Schema()
+export class Comment {
+    
+    @Prop({ required: true })
+    @unique()
```
Comment (bartlomiejmont) [inline ugbook-back/src/schemas/comment.schema.ts]: One author should be able to write multiple comments. So unique is not a good idea.

File: ugbook-back/src/schemas/comment.schema.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    authorId: string;
```
Comment (bartlomiejmont) [inline ugbook-back/src/schemas/comment.schema.ts]: authorId can be a string type but I think objectId would be a better choice.

**Review by bartlomiejmont — CHANGES_REQUESTED** [review]

File: ugbook-back/src/schemas/comment.schema.ts (hunk 1 above)
Comment (Darnoker) [reply ugbook-back/src/schemas/comment.schema.ts]: Corrected.

File: ugbook-back/src/schemas/comment.schema.ts (hunk 2 above)
Comment (Darnoker) [reply ugbook-back/src/schemas/comment.schema.ts]: Corrected.

**Review by bartlomiejmont — APPROVED** [review]

**Pull request merged**